examples/encrypted-communications/1.js

```diff
@@ -43,7 +42,8 @@ const createNode = async () => {
     )
   })
 
-  const { stream } = await node1.dialProtocol(node2.peerInfo, '/a-protocol')
+  const node2Multiaddr = `${node2.transportManager.getAddrs()[0]}/p2p/${node2.peerId.toB58String()}`
```

Comment:
In this instance I would store the multiaddrs in the address book and then dial the peer id, because the peer should have been organically discovered and this would have happened.